docs/sources/docker-hub/repos.md

```diff
@@ -1,42 +1,37 @@
-page_title: Repositories and images on Docker Hub
-page_description: Repositories and images on Docker Hub
+page_title: Your Repositories on Docker Hub
+page_description: Your Repositories on Docker Hub
 page_keywords: Docker, docker, registry, accounts, plans, Dockerfile, Docker Hub, webhooks, docs, documentation
 
-# Repositories and images on Docker Hub
+# Your Hub repositories
 
-![repositories](/docker-hub/hub-images/repos.png)
-
-## Searching for repositories and images
-
-You can `search` for all the publicly available repositories and images using
-Docker.
-
-    $ docker search ubuntu
+Docker Hub repositories make it possible for you to share images with co-workers,
+customers or the all Docker users. If you're building your images internally,
```

Comment:
typo, "all" and "the" are flipped. But it might be better to refer to "the Docker community at large" since "users" seems limited.